package.json

```diff
@@ -54,15 +54,15 @@
     "mozilla",
     "shield-study"
   ],
-  "license": "MIT",
+  "license": "MPL-2.0",
```

Comment:
I bravely changed this to MPL-2.0 since we have that LICENSE file at https://github.com/biancadanforth/tracking-protection-shield-study/blob/master/addon/LICENSE

Reply:
So brave!